config/environments/production.rb

```diff
@@ -42,11 +42,11 @@
   # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for NGINX
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  # config.force_ssl = true
+  # config.force_ssl = false
 
   # Use the lowest log level to ensure availability of diagnostic information
   # when problems arise.
-  # config.log_level = :debug
+  config.log_level = :info
```

Comment:
This is not related with the PR, plus it has just been fixed on master. Can you rebase against master and update your PR?